### PR TITLE
Fixed version update

### LINKED ISSUE #399: Update version does not update version number wpseo-news.php
### What did you expect to happen?
The version should be updated
### What happened instead?
It dit not get updated.

### PR DESCRIPTION
**Pull request by dutchmartin**
## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the grunt version-update would not update the version number in `wpseo-news.php`.
* Introduces the use of the yoast-grunt tasks grunt plugin.

## Relevant technical choices:

* Using the github url in package.json

## Test instructions

The Grunt commands set-version and update-version is the main way to change the version numbers over the various files in the plugin.

This PR can be tested by following these steps:

1. running `yarn`
2. Running `grunt set-version   --new-version=1`
3. Running `grunt update-version`
verify if the version is set to the argument given on step 2 in the following files:
    * README.md
    * wpseo-news.php
    * classes/class-wpseo-news.php

Fixes #399

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -2,9 +2,9 @@
 const { flattenVersionForFile } = require( "./grunt/modules/version.js" );
 
 module.exports = function(grunt) {
-	'use strict';
+	"use strict";
 
-	require('time-grunt')(grunt);
+	require("time-grunt")(grunt);
```
Comment (jcomack) [inline Gruntfile.js]: Missing spaces.

**Comment by jcomack** [comment]
Can you please fix your changelog entries? There's a typo and it's not written according to the standards that can be found here: https://github.com/Yoast/Wiki/wiki/Writing-issues-and-pull-requests#pull-requests

Additionally, the test steps vary from how it usually needs to be written. Please keep this in mind the next time you write them.

**Comment by Dieterrr** [comment]
CR done 👍

File: grunt/config/update-version.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ module.exports = {
     },
     pluginFile: {
         options: {
-            regEx: /(Version: )(\d+(\.\d+){0,3})([^\n^\.\d]?.*?)(\n)/,
+            regEx: /(Version: {1,10})(\d+(\.\d+){0,3})([^\n^\.\d]?.*?)(\n)/,
```
Comment (moorscode) [inline grunt/config/update-version.js]: This solution does not seem to be a real one.
Allowing for 1 to 10 spaces?

See the solution used in Yoast SEO
https://github.com/Yoast/wordpress-seo/blob/trunk/grunt/config/update-version.js#L18

**Comment by moorscode** [comment]
I have released the 1.0.0 version of the grunt plugin on NPM.
This should be used as the dependency instead of the Github repository.

**Pull request merged**